drivers/hydrophones/scripts/hydrophones

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
I think credit for these legacy sonar pkgs goes to Matt Thompson.  How do I change the author?